src/sagemaker/tuner.py

```diff
@@ -375,12 +375,7 @@ def _prepare_static_hyperparameters(
     ):
         """Prepare static hyperparameters for one estimator before tuning."""
         # Remove any hyperparameter that will be tuned
-        static_hyperparameters = {
-            str(k): str(v)
-            if not isinstance(v, (PipelineParameter, PipelineJsonGet, PipelineJoin))
-            else v
-            for (k, v) in estimator.hyperparameters().items()
-        }
+        static_hyperparameters = {str(k): str(v) for (k, v) in estimator.hyperparameters().items()}
```

Comment:
Sure! will do

Reply:
> update the Estimator implementation as well.

Has updated the Estimator in the new commit.

> double check the environment variables for processors and transformers.

Created a separate task to track as it will take me some time to understand and check. Will prioritize the test mechanism implementation at this moment.